pages/help_dialog.js

```diff
@@ -124,7 +124,17 @@ const HelpDialog = {
             });
           }
 
-          $$(descriptionElement, ".vimiumHelpDescription").textContent = command.description;
+          const MAX_LENGTH = 50;
+          const desiredOptionsLength = Math.max(0, MAX_LENGTH - command.description.length - 3);
```

Comment:
Find the length that the options list should be to make the full description the right length. Don't ever let it be less than 0, and leave room for the ellipsis "...".

Since this might not be clear, I will explain why I implement it this correct way. It bothers me how many people implement this wrong. Leaving room means that we fit as long lines as we can. For example, if we want a line fewer than 10 characters, not leaving room (not having the `- 3`) would mean we need to truncate anything over 7 characters so that the ... does not make the line too long. So we end up with:

"truncated!" -> "truncat..."

There is no reason to do this since the word could have fit on the line (word is 10 chars and we are fine with 10 chars). The other common wrong implementation is to truncate anything >= 10 and add the "..." giving:

"truncated!" -> "truncated!..."

This gives us a line that is **longer** than 10 chars!

The correct implementation (like this where we leave space) means only lines that need to be truncated are, and we never make a line too long:

"truncated!" -> "truncated!" (10 chars)
"even longer" -> "even lo..." (10 chars)

I hope this explains what the `-3` is for. I thought about adding a quick comment in the code there since it might not be clear, and I will do so if you like.